Makefile

```
@@ -530,7 +530,12 @@ bundle: manifests kustomize
 .PHONY: bundle-rhmi
 bundle-rhmi: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMAGE)
-	$(KUSTOMIZE) build config/manifests-rhmi | operator-sdk generate bundle -q --overwrite --version $(TAG) $(BUNDLE_METADATA_OPTS)
+	$(KUSTOMIZE) build config/manifests-rhmi | operator-sdk generate bundle -q --output-dir ./bundles/managed-api-service/$(TAG) --version $(TAG) $(BUNDLE_METADATA_OPTS)
```

I think the `--output-dir` should be `./bundles/integreatly-operator/$(TAG) `

```suggestion
	$(KUSTOMIZE) build config/manifests-rhmi | operator-sdk generate bundle -q --output-dir ./bundles/integreatly-operator/$(TAG) --version $(TAG) $(BUNDLE_METADATA_OPTS)
```

---

Thanks Carl! Good spot! (these are coming from my initial idea on how this could be approached)